Modify content (Write Example) (#531)
heremaps/here-olp-sdk-cpp: docs/dataservice-write-example.md

   ```cpp
   const std::string kKeyId("");            // your here.access.key.id
   const std::string kKeySecret("");        // your here.access.key.secret
   const std::string gCatalogHRN("");       // the HRN of the catalog to which you to publish data
   const std::string gLayer("");            // the of the layer inside the catalog to which you want to publish data
   ```

## Build and Run on Linux

To build and run the example on Linux:

1. Enable examples of the `CMake` targets.

   ```bash
   mkdir build && cd build
   cmake -DOLP_SDK_BUILD_EXAMPLES=ON ..
   ```

2. In the **build** folder, build the example.

   ```bash
   cmake --build . --target dataservice-write-example
   ```

3. Execute the example.

   ```bash
   ./examples/dataservice-write/dataservice-write-example
   ```

After building and running the example, the following message displays automatically: "Publish Successful - TraceID: \<TraceId generated by the platform>".

## Build and Run on Android

> To integrate the HERE OLP SDK for C++ libraries in the Android project:

Reviewer: ```suggestion
To integrate the HERE OLP SDK for C++ libraries in the Android example project:
```

Author: Fixed

- [Set up prerequisites](#prerequisites-android)
- [Build the HERE OLP SDK for C++](#build-sdk-android)
- [Build and Run the APK](#build-and-run-android)

### <a name="prerequisites-android"></a>Prerequisites

> Before you integrate the HERE OLP SDK for C++ libraries in your Android project:

Reviewer: ```suggestion
Before you integrate the HERE OLP SDK for C++ libraries in the Android example project:
```

Author: Fixed

3. If you want to build the SDK for a specific Android platform, use the `-DANDROID_PLATFORM CMake` flag, and if you want to build the SDK for a specific Android architecture, use the `-DANDROID_ABI` flag.
   For more details, see [NDK-specific CMake variables](https://developer.android.com/ndk/guides/cmake#variables).

   ```bash
   mkdir build && cd build
   cmake .. -DOLP_SDK_BUILD_EXAMPLES=ON -DCMAKE_TOOLCHAIN_FILE=$NDK_ROOT/build/cmake/android.toolchain.cmake -DANDROID_ABI=arm64-v8a
   ```

   The `CMake` command generates a `Gradle` project in the `build/examples/dataservice-write/android` folder.

4. Install the HERE OLP SDK for C++ libraries into the **sysroot** directory.

   ```bash
   # If necessary, execute as sudo.
   (sudo) make install
   ```

### <a name="build-and-run-android"></a>Build and Run the APK

To build and run the APK:

1. In the Android Studio IDE, open the `build/examples/dataservice-write/android/build.gradle` script.
2. Provide your application access key ID, access key secret, catalog HRN, and layer name.
3. Install and run the `dataservice_write_example` APK.

The main screen displays the following message: "Publish Successful".

> If you encounter an error, in your device LogCat, find the error message.

Reviewer: ```suggestion
If you encounter an error, find the error message in your device Logcat.
```
Refer to my previous comment in other PR: https://github.com/heremaps/here-olp-sdk-cpp/pull/529/files#r361130896

Author: Deleted

## Building and running on iOS

> To integrate the HERE OLP SDK for C++ libraries in a basic iOS application written in the Objective-C language:

Reviewer: ```suggestion
To integrate the HERE OLP SDK for C++ libraries in the iOS example application written in the Objective-C language:
```

Author: Fixed
